CHANGELOG.md

```diff
@@ -1,12 +1,30 @@
 # Changelog
 
-## unreleased
+## 3.4.0-rc.0 / 2025-04-29
 
 * [CHANGE] Make setting out-of-order native histograms feature (`--enable-feature=ooo-native-histograms`) a no-op. Out-of-order native histograms are now always enabled when `out_of_order_time_window` is greater than zero and `--enable-feature=native-histograms` is set. #16207
 * [FEATURE] OTLP translate: Add feature flag for optionally translating OTel explicit bucket histograms into native histograms with custom buckets. #15850
 * [FEATURE] OTLP translate: Add option to receive OTLP metrics without translating names or attributes. #16441
+* [FEATURE] Promltest: support to test expectation annotations. #15995
```

Comment:
would this be exposed to users via `promtool`? if yes we can use `Promtool:`, otherwise we can omit it I think.

Reply:
```suggestion
```